corehq/apps/fixtures/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class Config(AppConfig):
+    name = 'corehq.apps.fixtures'
+    default_auto_field = 'django.db.models.BigAutoField'
```

Comment:
seems like this isn't actually relevant to the models since they are either using `AutoField` or `UUID`

Reply:
It's used by the implicit `id` of `LookupTableRowOwner`. I also want any new models in this app to default to this if not otherwise specified.